gax/test/showcase-echo-client/README.md

```diff
@@ -1,12 +1 @@
-## showcase-echo-client
-
-This folder contains a client library for the `Echo` service of
-[gapic-showcase](https://github.com/googleapis/gapic-showcase),
-as generated by
-[gapic-generator-typescript](https://github.com/googleapis/gapic-generator-typescript).
-
-It uses `google-gax`, so by having this generated code here, we can verify that the
-changes made to this library don't break the code of the generated libraries.
-
-The only modification is in `package.json`, where it's set to depend on the local
-version of `google-gax` being tested.
```

Comment:
@sofisl do we need to add this to some kind of template so it doesn't get overwritten?

Reply:
oh good call! I accidentally overwrote it. I'll revert now.